src/commands/init.ts

```diff
@@ -46,11 +46,15 @@ export class InitCommand extends Command {
       outputDir: args.output,
     };
 
-    checkDirectoryIsEmpty(config.outputDir);
+    InitCommand.validateConfig(config);
 
     return config;
   };
 
+  static validateConfig = (config: InitCommandConfig): void => {
+    checkDirectoryIsEmpty(config.outputDir);
+  };
+
```

Comment:
Is this refactor to allow us to perform more actions to validate config in the future?

Reply:
In part, yes. It separates the concerns rather than doing both in `getConfig` which hopefully improves the readability and also makes it easier to test.